src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -390,8 +391,9 @@ bool emberAfGroupsClusterRemoveAllGroupsCallback(EndpointId aEndpoint, app::Comm
                 }
                 else
                 {
-                    uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
-                    GroupId groupId                                               = binding.groupId;
+                    GroupId groupId                                                 = binding.groupId;
+                    uint8_t groupNameBuffer[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH] = { 0 };
+                    chip::ByteSpan groupName(groupNameBuffer);
```

Comment:
Again, this does not make sense to me.  We used to pass in an empty name, and now we are passing in a decidedly non-empty one, but full of null chars.